feat(content): Add Journalist flavour mission (#6605)
endless-sky/endless-sky: data/human/culture conversations.txt

			`	Moments later the small crowd allows him to depart opposite your viewpoint, without incident and without explanation. You aren't quite sure what to make of this scene as the bodies are swiftly dragged away.`
				decline

mission "News Reporter - Paradise"
	minor
	source
		government "Republic"
		attributes "paradise"
		not attributes "station"
	to offer
		random < 8
		has "event: war begins"
		not "event: joined the free worlds"
	on offer
		conversation

> 			`You are passing through the spaceport when you see a young woman that is walking in your direction. You realize she is a news reporter based on th multitude of camera drones trailing behind her.`

Reviewer: Changed phrasing.
```suggestion
			`You are passing through the spaceport when you see a young woman walking in your direction. Trailing behind her is a multitude of camera drones, marking her as a news reporter.`
```

Reviewer 2: I'm of the camp that shows the reader that they're a news reporter rather than tell. Here, a multitude of camera drones is already an oddity. I'd say that having them marked as news drones or in a colour that doesn't reflect light would be good.

Reviewer: Updated my suggestion.

			choice
				`	(Hope she doesn't notice you.)`
					goto walk
				`	(Try to avoid her.)`

> 			`	You take a hallway off to the side to evade her. Just as you think you have gotten out of sight you hear a voice. "Excuse me!" You turn to see the reporter following you.`

Reviewer: ```suggestion
			`	You take a hallway off to the side to evade her. Just as you think you've gotten out of sight, you hear a voice. "Excuse me!" You turn to see the reporter following you.`
```

			label walk

> 			`	As she gets closer to you, she smiles. "Hello captain! Would you be willing to answer a few questions for our viewers?"`

Reviewer: ```suggestion
			`	As she gets closer to you, she smiles. "Hello Captain! Would you be willing to answer a few questions for our viewers?"`
```

			choice

> 				`	"Sure, I don't mind answering some questions."`

Reviewer: Remove repetition.
```suggestion
				`	"Sure, I wouldn't mind."`
```

					goto questions

> 				`	"No, I would rather not."`

Author: It does sound better! I don't use contractions as much as some do, so that is probably why I write that way!

					decline
			label questions

> 			`	The camera drones circle around you as she prepares for the interview. In a few seconds, she smiles at one of the drones and begins.`

Reviewer: ```suggestion
			`	The camera drones circle around you as she prepares for the interview. After a few seconds, she smiles at one of the drones and begins.`
```

			`	"Welcome back, I'm here on <origin> reporting on the situation with the Free Worlds coalition. I'm joined by a merchant captain that is between trips. Captain, what do you think of the ultimatum sent to Parliament?"`
			choice
				`	"I think Parliament should reconsider and give the situation some thought."`
					goto reconsider
				`	"Parliament made the right decision by refusing the demands."`
					goto right
			label reconsider

> 			`	"Can you explain your answer for our viewers?" she asks.`

Reviewer: Minor nitpick, but I think this mission would be improved if the reporter reacted more to what the player says, even if it's subtle. Maybe something like having the reporter tighten up on one answer and relax on the other.

Author: I'm not currently at my computer, so I will take a look at it and come up with something.

Author: I did adjust it a little bit, hopefully that flows better. I feel that most of the people in the Paradise Worlds would be against the Free Worlds, so I wrote the journalist the same way.

			choice

> 				`	"The interests of every world should be considered whether they agree or not."`

Author: Yes, it is. Thank you for the suggestions.

		conversation
			`You are passing through the spaceport when you see a young woman walking in your direction. Trailing behind her is a multitude of camera drones, each one marked with a logo consisting of two curves pointing inwards.`
			choice
				`	(Wait for her to approach.)`
					goto walk
				`	(Hope she doesn't notice you.)`
					goto notice
				`	(Try to avoid her.)`

> 			`	You take a hallway off to the side to evade her. Just as you think you've gotten out of sight, you hear a voice. "Excuse me!" You turn to see the reporter following you.`

Reviewer 3: ```suggestion
			`	You take a hallway off to the side to evade her. Just as you think you've gotten out of sight, you hear a voice. "Excuse me!" You turn to see the reporter following you.`
				goto walk
```
